doc/satabs-user-manual.md

```diff
@@ -57,7 +57,7 @@ and the directories that contain the header files. You must run SATABS
 from within the *Visual Studio Command Prompt*.
 
 Note that the distribution files for the [Eclipse
-plugin](installation-plugin.shtml) include the command-line tools.
+plugin](http://www.cprover.org/eclipse-plugin/) include the command-line tools.
```

Comment:
Is this still even slightly run-able?  I remember trying to use it in 2016 and it seemed it had bit-rotted.

Also, I love archiving and documentation but given the current status of satabs ( https://github.com/diffblue/cbmc/pull/7476 ), should this still be in the manual?

Reply:
Don't shoot the messenger :-)  - I have created https://github.com/diffblue/cbmc/issues/7478.